apps/prairielearn/src/pages/instructorAssessmentInstance/instructorAssessmentInstance.ejs

```diff
@@ -40,6 +40,53 @@
             <tr><th>Instance</th><td colspan="2"><%= assessment_instance.number %>
                 (<a href="<%= plainUrlPrefix %>/course_instance/<%= course_instance.id %>/assessment_instance/<%= assessment_instance.id %>">student view</a>)
             </td></tr>
+            <% if (instance_user) { %>
+            <tr>
+              <th>Fingerprint Changes</th>
+              <td colspan="2">
+                <%= assessment_instance.client_fingerprint_id_change_count %>
+                <% if (assessment_instance.client_fingerprint_id_change_count > 0) { %>
+                  <div class="btn btn-xs" role="button"
+                      data-toggle="modal" data-target="#fingerprintChangeModal"><i class="fa fa-question-circle"></i></div>
```

Comment:
I'm not sure we need this modal. Isn't it basically showing the same info as in the Log table below? Instead I think we should have a modal that explains what Fingerprint Changes means and when to be worried.

Reply:
You are correct. The information here seemed redundant with the info in the event logs. I have now updated this to display a popover that explains fingerprint changes.